Competition2018/src/main/java/competition/subsystems/drive/commands/DriveForDistanceCommand.java

```diff
@@ -101,7 +101,7 @@ public boolean isFinished() {
     @Override
     public void end() {
         log.info("Ending, PreviousPosition was " + startingPosition + ", Targeted Delta in distance is " + deltaDistance
-                + " Distance traveled is " + (deltaDistance - startingPosition));
```

Comment:
`", Current position should be " + (deltaDistance + startingPosition));`

Reply:
Why? I would prefer having it print the target distance and the actual distance, as it was previously. Deltas are easier to compare than absolutes.

Reply:
@WasabiFan 

With the current changes, it's printing:
* Starting position
* Delta distance
* Final position

Before, it was printing:
* Starting position
* Delta distance
* the difference between delta distance and final position (a meaningless number?)

Reply:
Ah, got it; this is the same math that was in the original PR. In that case, the preferred calculation would be `current - initial`.